include/wx/setup_inc.h

```diff
@@ -1467,6 +1467,9 @@
 // Set to 1 for TIFF format support (requires libtiff)
 #define wxUSE_LIBTIFF       1
 
+// Set to 1 for SVG rasterizing support using LunaSVG (C++17 compiler required)
+#define wxUSE_LUNASVG       0
```

Comment:
How comes that NanoSVG is the default here as advertised in the docs, but LunaSVG seems to be the default in the actual `setup.h` files?

By default, it seems `OFF` in CMake and `1` in `config.{gcc|vc)`?